### PR TITLE
Add artifacts naming doc

### PR DESCRIPTION
**Pull request by mauromorales**
:warning: let's merge this until all the changes to the artifacts are done, so we don't have to open fixes in case something changes

Relates to https://github.com/kairos-io/kairos/issues/1109

### CODE REVIEW

File: content/en/docs/Reference/artifacts.md
Comment (Itxaka) [inline content/en/docs/Reference/artifacts.md:32]: The mix of arm64 and aarch64 is very confusing, aren't they the same?

File: content/en/docs/Reference/artifacts.md
Comment (Itxaka) [inline content/en/docs/Reference/artifacts.md:41]: Missing mentioning the img repos

**Review by Itxaka — APPROVED** [review]

File: content/en/docs/Reference/artifacts.md
Comment (mauromorales) [reply content/en/docs/Reference/artifacts.md:26]: yeah it's not the best I agree, but unfortunately we have repeted extensions, e.g. trivy and sbom both have json ... would something like `.trivy.json` and `.sbom.json` make more sense?

File: content/en/docs/Reference/artifacts.md
Comment (mauromorales) [reply content/en/docs/Reference/artifacts.md:32]: yeah good point, will fix ... also I see we have in some places x86_64 but translate to amd64, do you know what is prefered?

File: content/en/docs/Reference/artifacts.md
Comment (mauromorales) [reply content/en/docs/Reference/artifacts.md:41]: the optional `[-<extension>]` would include this

File: content/en/docs/Reference/artifacts.md
Comment (Itxaka) [reply content/en/docs/Reference/artifacts.md:26]: Meh, that sucks but it's ok

File: content/en/docs/Reference/artifacts.md
Comment (Itxaka) [reply content/en/docs/Reference/artifacts.md:32]: Depends on the day and location lol.

Kernel reports x86 and aarch64, golang says x86 and amd64 and goreleaser does a mix I think

We just need one and use it over everywhere

Comment (mauromorales) [reply content/en/docs/Reference/artifacts.md]: Ok, went with the golang version, just because it's documented in one place but also left a notice at the beginning of the document so users can be aware of this

**Review by Itxaka — APPROVED** [review]

**Review by jimmykarily — APPROVED** [review]

**Comment by jimmykarily** [comment]
We can merge this PR after the implementation is done: https://github.com/kairos-io/kairos/issues/1109

File: content/en/docs/Reference/artifacts.md
Comment (mauromorales) [inline content/en/docs/Reference/artifacts.md:104]: @jimmykarily according to the discussion today we will use tags for the support of the image. Do you know how that looks in the end? Maybe something like:

`quay.io/kairos/core-ubuntu-generic:community-v1.0.0`

and if that is the case, should we also move the model into the tag?

`quay.io/kairos/core-ubuntu-generic:community-generic-v1.0.0`

Comment (jimmykarily) [reply content/en/docs/Reference/artifacts.md:104]: Not "tags", "labels". Like these: https://docs.docker.com/engine/reference/builder/#label

**Comment by mauromorales** [comment]
This is just waiting for a release with the new artifacts names

**Comment by Itxaka** [comment]
This is done now!

**Pull request merged**
